### DISCUSSION TITLE
How to skip processing of repositories with open onboarding pull requests

### DISCUSSION THREAD

**Discussion by dmosen** [renovatebot/renovate#16489] [Request Help] [answered]
Hi,

We are running a self-hosted renovate installation and the runtime of our renovate jobs is quite long and we'd like to reduce it.

I wonder if it is possible to skip the processing of repositories if there is still an open onboarding/configuration pull request. As far as I can see, the runtime for processing a configured repository is nearly the same as for an unconfigured one. Having the option to skip processing as stated above would greatly reduce the runtime in our current situation.

Best regards,
Dominik

**Comment by StinkyLord** [comment]
if i understand correctly, you are looking to skip specific repositories? 
the option is: [This](https://docs.renovatebot.com/self-hosted-configuration/#autodiscoverfilter)

i don't think you can tell renovate to not process any repo that is NOT onboarded.. it will take a look at it anyway to see if there are any changes made, unless you filter it out using your repositories defined in your `config.ts` or in your filtered `autodiscoverfilter`

**Comment by rarkins** [comment, marked as answer]
There doesn't exist an option for this today. You want it to just skip processing the repo as soon as it finds that there's an onboarding PR still open? It will still take some time, but not as much as doing a full processing.

**Reply by dmosen** [reply]
@rarkins Yes, that's exactly what I meant. Do you think it would be worth implementing such an option or is it an odd use case?

**Reply by joergsesterhenn** [reply]
@rarkins That sounds like a simple enough solution and could have great impact on the speed (no looking for/parsing of files necessary) of renovate at the very low cost of outdated "what to expect"-Information, which has little value in my opinion. 

We'd use this feature if it was implemented and I bet others might find it useful as well.

Also, are there any other tips on speeding up renovate for a large number of repositories apart from running filtered subsets in parallel?
